engine/components-api/src/main/java/com/cloud/vm/snapshot/VMSnapshotManager.java

```diff
@@ -45,7 +66,7 @@
      * Sync VM snapshot state when VM snapshot in reverting or snapshoting or expunging state
      * Used for fullsync after agent connects
      *
-     * @param vm, the VM in question
+     * @param vm,    the VM in question
```

Comment:
since you are fixing the java doc; pleae remove the comma.

Reply:
@DaanHoogland Done